src/visualization/RunTopSheet.vue

```diff
@@ -128,7 +128,8 @@ export default {
 }
```

Comment:
I din't bother adapting this component since it is not used. @billyc can we remove it?